src/main/scala/me/rexim/issuestant/Main.scala

```diff
@@ -1,13 +1,32 @@
 package me.rexim.issuestant
 
+import me.rexim.issuestant.github.EventsSource
+
+import scala.concurrent.duration.{FiniteDuration, SECONDS}
+
 import scalaz.concurrent.Task
 
 import org.http4s.server.{Server, ServerApp}
 import org.http4s.server.blaze._
+import org.http4s.client.blaze._
 
 object Main extends ServerApp {
-  // TODO: Use the HTTP port passed by heroku
+  def pollingLoop(permalink: Permalink): Unit = {
+    permalink.update.flatMap { nextPermalink =>
+      Task.schedule(pollingLoop(nextPermalink), FiniteDuration(1, SECONDS))
+    }.runAsync { _ => }
+  }
+
+  // TODO(19f86d8b-7c7f-4acc-af5b-a3a4c29cec5c): Use the HTTP port passed by heroku
   override def server(args: List[String]): Task[Server] = {
+    Task.schedule(pollingLoop(new Permalink(
```

Comment:
Extract this polling loop into a separate class. That will help you with testing it and meeting the coverage threshold.